week-3/C-array-every/exercise.js

```diff
@@ -5,7 +5,13 @@
 var students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 var group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-var groupIsOnlyStudents; // complete this statement
+var groupIsOnlyStudents = group.every(checkGroup); // complete this statement
+
+function checkGroup(students, group) {
+  if ([group] === [students]) {
```

Comment:
Review what every() function expects as its parameter. What are the input and output to the checkGroup function here?